quodlibet/qltk/queue.py

```diff
@@ -345,7 +345,35 @@ def __queue_disable(self, disabled):
 
 
 class QueueModel(PlaylistModel):
-    """Own class for debugging"""
+    """
+    A play list model for queues.
+
+    In contrast to `PlaylistModel`, when new songs have been set, its play order
+    will disregard the current song from the old set of songs and start from
+    scratch.
+    """
+
+    __reset = False
+
+    def set(self, songs: Sequence[Any]):
+        self.__reset = True
+        super().set(songs)
+
+    def next(self):
+        print_d("Using %s.next_explicit() to get next song" % self.order)
```

Comment:
```suggestion
        print_d(f"Using {self.order} to get next song")
```
(we're moving slowly to f-strings everywhere)